control-plane/config/post-install/200-controller-cluster-role.yaml

```diff
@@ -0,0 +1,40 @@
+---
+
+# Copyright 2022 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: knative-kafka-controller-post-install
+  labels:
+    kafka.eventing.knative.dev/release: devel
+rules:
+  - apiGroups:
+      - apps
+    resources:
+      - deployments
+    verbs:
+      - delete
+      - deletecollection
+
+  # sources.knative.dev resources and finalizers we care about.
+  - apiGroups:
+      - sources.knative.dev
+    resources:
+      - kafkasources
+      - kafkasources/status
+    verbs:
```

Comment:
are these verbs enough?

Reply:
Yes, these are enough

- list Kafka sources: https://github.com/knative-sandbox/eventing-kafka-broker/pull/1889/files#diff-736787d0a0ff25859f1fa32f6413a911418e4676d53deddb52980359818eaa24R42-R45
- and then delete collection: https://github.com/knative-sandbox/eventing-kafka-broker/pull/1889/files#diff-736787d0a0ff25859f1fa32f6413a911418e4676d53deddb52980359818eaa24R64-R67